### ISSUE TITLE
Icon request: bed

### ISSUE THREAD

**Issue by jonathanrd** [colebemis/feather#68]
Hi, thanks for the great set of icons!

How about some icons for describing rooms within a house

**Icon name:** bed
**Use case:** Many real estate agents make use of such icons when advertising properties.

I've included an example below.

![image](https://cloud.githubusercontent.com/assets/3124180/26781561/18018056-49e7-11e7-80e4-ad6fe9c543e5.png)

**Comment by pkrawc** [comment]
[Feather Compliant Bed Icon](https://www.figma.com/file/VN8Ma3Uf1ewUvRvGdRAkMv6b/bed?node-id=2%3A2)

@colebemis @jonathanrd

**Comment by johnletey** [comment]
@pkrawc It looks as though the bed as a hole at the top 😆 ... other than that looks pretty good!

@locness3 What do you think?

**Comment by locness3** [comment]
I agree with @johnletey. Also, please make sure to follow #171 before saying it is "Featehr Compliant".

**Comment by pkrawc** [comment]
@locness3 I did follow #171. That's what lead me to say it's complaint.

The bed loses visual interest if you don't differentiate the headboard.

**Comment by locness3** [comment]
> The bed loses visual interest if you don't differentiate the headboard.

I don't think so. Feather Icons are simple icons, without visual interest, just looking beautiful

**Comment by johnletey** [comment]
Well said @locness3!

**Comment by pkrawc** [comment]
@locness3 visual interest = looking beautiful. So I disagree, but you too seem to be in accordance.

**Comment by pkrawc** [comment]
I've updated this icon and opened a pull request

@johnletey @locness3

**Comment by locness3** [comment]
Let's say they are beautiful by their simplicity

**Comment by dbedoyat** [comment]
![image](https://user-images.githubusercontent.com/17677474/198916020-bb09e996-a6ba-4ab4-94e8-697230f92e81.png)

I made a better suggestion, let me know if you have feedback.
Here is the SVG code:

`<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M2 20V16.3334M2 16.3334V12.0001C2 12.0001 2 9.33333 4.35294 9.33341C6.70588 9.3335 17.2941 9.3335 19.6471 9.33341C22 9.33333 22 12.0001 22 12.0001V16.3333M2 16.3334L22 16.3333M22 20V16.3333M4 9C4 7.93333 3.17647 4 6.70588 4C10.2353 4 13.7647 4 17.2941 4C20.8235 4 20 9 20 9" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"/>
</svg>
`

**Comment by karsa-mistmere** [comment]
Nice, virtually identical to one of the multiple bed icons we have at Lucide: https://lucide.dev/icon/bed-double?search=bed

**Comment by dbedoyat** [comment]
> Nice, virtually identical to one of the multiple bed icons we have at Lucide: https://lucide.dev/icon/bed-double?search=bed

Fantastic, I didn't know a cool fork for feather icons. Are you open to contributions?

**Comment by locness3** [comment]
> Are you open to contributions?

Yes, they are
